<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarchingCubesParamsDialog</class>
 <widget class="QDialog" name="MarchingCubesParamsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Marching Cubes </string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ISO level:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="isoLevelSpinBox">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Grid resolution (x):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="gridResolutionXSpinBox">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Grid resolution (y):</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Grid resolution (z):</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="gridResolutionYSpinBox">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="gridResolutionZSpinBox">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Grid extension percentage:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="gridExtensionPercentageSpinBox">
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
      <property name="value">
       <double>0.200000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>MarchingCubesParamsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
